app/controllers/caseload_controller.rb

```diff
@@ -22,7 +22,29 @@ def new
 private
 
   def sort_allocations(allocations)
-    sort_collection(allocations, default_sort: :last_name)
+    if params['sort'].present? && params['sort'].downcase.include?('cell_location')
```

Comment:
I think you should be good to use the 'sort_params' method here to pick up the parameters rather than grabbing them by hand